distance_metrics.py

```diff
@@ -0,0 +1,284 @@
+# Copyright (C) 2020 and later: Google, Inc.
+
+import cv2
+import enum
+import numpy as np
+
+class ImgFormat(enum.Enum):
+    """Enumeration class for image format."""
+    RGB = 1 # RGB format, necessary for ClearType
```

Comment:
Why do we need to support 4 different image formats? Are you using all 4 different formats in your project?

Reply:
I am using 2 formats. Also want to leave metrics for future experiments.

Reply:
So, which two are needed in the end.

Reply:
ImgFormat.RGB and ImgFormat.EMBEDDINGS